.vscode/settings.json

```diff
@@ -1,11 +0,0 @@
-{
-  "eslint.validate": [
-    "javascript",
-    "javascriptreact",
-    {
-      "language": "vue",
-      "autoFix": true
-    }
-  ],
-  "editor.formatOnSave": false,
-}
```

Comment:
I wouldn't want to introduce this change... can you just run `git checkout origin/master -- .vscode/settings.json` from the root path?